noir/aztec_macros/src/lib.rs

```diff
@@ -343,9 +386,16 @@ fn transform_module(
 
     // Check for a user defined storage struct
     let storage_defined = check_for_storage_definition(module);
+    let storage_implemented = check_for_storage_implementation(module);
+
+    let crate_graph = &context.crate_graph[crate_id];
+
+    if storage_defined && !storage_implemented {
+        generate_storage_implementation(module)
+            .map_err(|err| (err.into(), crate_graph.root_file_id))?;
```

Comment:
This into can be removed! In general clippy is complaining on some points where we can remove bits of redundant code

Reply:
Reviewed all clipply warnings!